specs/UrbanMobility/GtfsStopTime/schema.json

```diff
@@ -46,6 +46,10 @@
             "0", "1", "2", "3"
           ]
         },
+		"stopHeadsign": {
```

Comment:
check indentation and content, this is a string ...

Reply:
Fixed in 63357ef